.goreleaser.yml

```diff
@@ -76,14 +76,15 @@ checksum:
 signs:
   -
     id: with_key_id
-    # TODO: Replace with variable once goreleaser supports variables here
-    signature: "${artifact}.72D7468F.sig"
-    args: ["--batch", "--no-tty", "-u", "{{ .Env.GPG_KEY_ID }}", "--output", "${artifact}.{{ .Env.GPG_KEY_ID }}.sig", "--detach-sign", "${artifact}"]
+    signature: "${artifact}.{{ .Env.GPG_KEY_ID }}.sig"
```

Comment:
Would it be possible to get the key id from signore, so it can be removed from our repository secrets?

Reply:
This is a request other folks have made, and a good call. I've been mildly mixed on how to handle this because we're identifying signatures in [releases](https://releases.hashicorp.com) with the key id of the primary key but signing with a subkey (which is trusted by the primary key). I can add something like `key_id` and `primary_key_id` to the signers metadata, I just don't want people to get confused and use the subkey key id.

The key id for the current HashiCorp Security key is [`72D7468F`](https://keys.openpgp.org/vks/v1/by-fingerprint/C874011F0AB405110D02105534365D9472D7468F).

Reply:
Thanks for the explanation! Having something like `key_id` and `primary_key_id` sounds useful.

I intend to reduce the number of things that need to be updated when a key expires or needs rotation. So, if we can avoid having a hard-coded value here, there would be one less file to update.

How does signore handle the expiration of subkeys? For example, do we need to create a new client in April to use a new signer with the new subkey? Or is signore taking care of this for us?

Reply:
My plan is to add a new signer for the "long-term" signing subkey and alias the current interim signing subkey signer to use that. The signore client will be changed to add a soft deprecation notice that users should update the signer in their config. We're planning on re-signing everything in the releases S3 bucket signed with the interim key with the long-term key before then as well.

So essentially, there shouldn't be any need for signore users to change anything come April, except to shush the deprecation notice.

Reply:
Great! Having an alias to the new signing subkey simplifies things.

I'll keep `72D7468F` in here until we can get the ID from signore. Thanks again for taking the time to explain things and reviewing the PR.